src/ChatData.json

```
@@ -0,0 +1,58 @@
+{   
+    "black" : [
+        {"isMe": false, "text": "hey wassup You to party tonight?"},
```

```suggestion
        {"userId": 0, "text": "hey wassup You to party tonight?"},
```
추후 단톡방 기능과 같이 확장성을 생각하면 userId를 이용해 본인 userID 와 비교해서 isMe 여부를 파악하는게 좋을 것 같아요.

---

저도 이 말에 동의합니다 ㅎㅎ